tests/kani/ThreadLocalRef/main.rs

```diff
@@ -9,9 +9,8 @@ thread_local! {
 
 #[kani::proof]
 fn main() {
-    COND.with(|&b|{
+    COND.with(|&b| {
```

Comment:
How did the formatter miss this?

Reply:
I think this test may have been committed before we had the format checker?

Reply:
The format checker runs in all files, not just the ones that has changed.

Reply:
I don't think `cargo fmt` formats everything in all subfolders. Opened an issue: #1455

Reply:
Thanks. Fixed here https://github.com/model-checking/kani/pull/1459